AnalysisReporting/analysisreport/models.py

```diff
@@ -0,0 +1,13 @@
+
+from django.contrib.auth.models import User
+from django.db import models
+
+# Create your models here.
+
+ 
```

Comment:
Please follow this for spacing between code blocks 
https://www.python.org/dev/peps/pep-0008/#blank-lines